src/contents/all/components/app/LoginForm.tsx

```diff
@@ -0,0 +1,101 @@
+/*global chrome*/
+
+import React, { useEffect, useState } from 'react';
+import Frame, { FrameContextConsumer } from 'react-frame-component';
+import { useDispatch, useSelector } from "react-redux";
+import { Link } from 'react-router-dom';
+import { hot } from 'react-hot-loader/root';
+import * as Sentry from '@sentry/browser';
+
+import { Form, Icon, Input, Button, message, notification } from 'antd';
+
+import './App.scss';
+
+//import './LoginForm.scss'
+
+interface Props {
+  form: any
+}
+
+const FormItem = Form.Item;
+
+const LoginForm = () => {
+
+  const {error, inProgress} = useSelector(state => ({ ...state.auth, ...state.common}))
```

Comment:
unsure about this from mainframe

Reply:
Yea this is all redux stuff we use specifically in mainframe, unnecessary here we can remove it.